beater/api/mux.go

```diff
@@ -179,15 +180,15 @@ type middlewareFunc func(*config.Config, *authorization.Handler, map[request.Res
 
 func agentConfigHandler(cfg *config.Config, authHandler *authorization.Handler, middlewareFunc middlewareFunc) (request.Handler, error) {
 	var client kibana.Client
-	if cfg.Kibana.Enabled() {
-		client = kibana.NewConnectingClient(cfg.Kibana)
+	if cfg.Kibana.Enabled {
+		client = kibana.NewConnectingClient(common.MustNewConfigFrom(cfg.Kibana))
```

Comment:
Can you please add a TODO (or just do it) to pass in the `kibana.ClientConfig` directly, rather than another round-trip through `libbeat/common`? Then we can change `apm-server/kibana` to call `libbeat/kibana.NewClientWithConfig`, rather than _yet another_ round-trip :)

Reply:
Good call, just did it